src/witan/models/dem/ccm/models.clj

```diff
@@ -17,7 +17,7 @@
 
    ;; asfr/asmr projections
    [:calc-hist-asfr :project-asfr]
-   [:calc-hist-asmr :project-asmr]
+   [:calc-hist-asmr :project-asmr-average-fixed]
```

Comment:
thanks, will change back in both places- was making changes until the tests passed and didn't initially pick up that this wasn't a necessary one!